lib/deploy/activate.rb

```diff
@@ -109,15 +110,18 @@ def download_from_s3_and_update_permissions(src, dest)
     end
 
     def download_file(src, dest)
-      ec2_region = ec2_data.region
-
-      Identity::Hostdata::S3.new(
-        bucket: "login-gov.secrets.#{ec2_data.account_id}-#{ec2_region}",
-        env: nil,
-        region: ec2_region,
-        logger: logger,
-        s3_client: s3_client,
-      ).download_configs(src => dest)
+      secrets_s3.download_file(
+        s3_path: src,
+        local_path: dest,
+      )
+    end
```

Comment:
let's just completely remove this method and inline it?

Reply:
because `download_file` is a generic name and it doesn't tell me *where* the file comes from